docs/root/configuration/http/http_filters/header_to_metadata_filter.rst

```diff
@@ -26,64 +26,29 @@ Example
 A sample filter configuration to route traffic to endpoints based on the presence or
 absence of a version header could be:
 
-.. code-block:: yaml
-
-  http_filters:
-    - name: envoy.filters.http.header_to_metadata
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.filters.http.header_to_metadata.v3.Config
-        request_rules:
-          - header: x-version
-            on_header_present:
-              metadata_namespace: envoy.lb
-              key: version
-              type: STRING
-            on_header_missing:
-              metadata_namespace: envoy.lb
-              key: default
-              value: 'true'
-              type: STRING
-            remove: false
+.. literalinclude:: _include/header-to-metadata-filter.yaml
+    :language: yaml
+    :lines: 25-40
```

Comment:
```suggestion
    :lines: 25-40
    :lineno-start: 25
```

same elsewhere otherwise the numbering starts from 1

Reply:
Fixed.